src/engines/nwn/gui/widgets/button.cpp

```diff
@@ -68,6 +69,16 @@ void WidgetButton::leave() {
 		_model->setState("up");
 }
 
+void WidgetButton::setPosition(float x, float y, float z) {
+	ModelWidget::setPosition(x, y, z);
+
+	// Dont write on the border
```

Comment:
Please spell it `Don't`, not `Dont`. This goes for the commit messages (76a516f258b2b4e8a97bd39bedc97ed45ca80928 and 4068f709a148335ef867aa226596257e4aee01a6) as well.